### PR TITLE
EOS-21409: removed enable from kibana and csm as pcs enables services

### PR DESCRIPTION
**Pull request by tanujashinde0405**
Signed-off-by: Tanuja Shinde <tanuja.shinde@seagate.com>

### CODE REVIEW

**Review by mazinamdar — APPROVED** [review]

**Review by 83bhp — APPROVED** [review]

**Review by nitin-seagate — APPROVED** [review]
LGTM

File: srv/components/misc_pkgs/kibana/start.sls [hunk 1]
```diff
@@ -18,4 +18,4 @@
 Start Kibana:
   service.running:
     - name: kibana
-    - enable: True
+    - enabled: True
```
Comment (nitin-seagate) [inline srv/components/misc_pkgs/kibana/start.sls:21]: Please check this one

**Review by nitin-seagate — CHANGES_REQUESTED** [review]

**Review by nitin-seagate — APPROVED** [review]
LGTM

File: srv/components/misc_pkgs/kibana/start.sls (hunk 1 above)
Comment (83bhp) [reply srv/components/misc_pkgs/kibana/start.sls]: enable is correct..

**Pull request merged**